dymos/examples/vanderpol/vanderpol_dymos.py

```diff
@@ -75,6 +78,11 @@ def vanderpol(transcription='gauss-lobatto', num_segments=8, transcription_order
     # setup the problem
     p.setup(check=True)
 
+    # TODO - Dymos API will soon provide a way to specify this.
+    # the linear solver used to compute derivatives is not working on MPI, so switch to Linear Block Gauss Seidel
+    for phase in traj._phases.values():
+        phase.linear_solver = om.LinearRunOnce()
```

Comment:
Inconsistency between comments and code here.  Comments say Linear Block Gauss Seidel (`om.LinearBlockGS`), but code uses `om.LinearRunOnce`.

Reply:
comment changed to match code